checkov/main.py

```diff
@@ -167,6 +167,11 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     elif config.bc_api_key:
         logger.debug(f'Using API key ending with {config.bc_api_key[-8:]}')
 
+        if not bc_integration.is_token_valid(config.bc_api_key):
+            raise Exception(f'The provided API key does not appear to be a valid Bridgecrew API key or Prisma Cloud '
+                            f'access key and secret key. For Prisma, the value must be in the form '
+                            f'ACCESS_KEY::SECRET_KEY')
```

Comment:
just remove the `f` before the strings here, then the linter is happy 😄

Reply:
I cant even remember why I initially added the f...